tests/test_models.py

```diff
@@ -37,32 +37,33 @@ def tearDownClass(cls):
     def setUp(self):
         """ This runs before each test """
         db.session.query(Customer).delete()  # clean up the last tests
+        db.session.query(Address).delete()
         db.session.commit()
 
     def tearDown(self):
         """ This runs after each test """
         db.session.remove()
 
-    ######################################################################
-    #  T E S T   C A S E S
-    ######################################################################
+    #####################################################################
+    #  C U S T O M E R   A N D   A D D R E S S   M O D E L   C A S E S  #
+    #####################################################################
 
     def test_create_customer(self):
         """ It should Create a Customer and assert that it exists """
-        customer = Customer(first_name="Marwan",last_name="J",email="mya6510@nyu.edu",password="12344321")
+        customer = Customer(first_name="Marwan",last_name="J",email="mya6510@nyu.edu",password="12344321", addresses=[])
         self.assertEqual(str(customer), "<Customer J, Marwan id=[None]>")
         self.assertTrue(customer is not None)
         self.assertEqual(customer.id, None)
         self.assertEqual(customer.first_name,"Marwan")
         self.assertEqual(customer.last_name, "J")
         self.assertEqual(customer.email, "mya6510@nyu.edu")
         self.assertEqual(customer.password, "12344321")
```

Comment:
self.assertEqual(customer.addresses, [])

Reply:
Thanks added the case.